tests/raw_query/tests.py

```diff
@@ -187,22 +181,32 @@ def test_order_handler(self):
         )
 
         for select in selects:
-            query = "SELECT %s FROM raw_query_author" % select
+            cols = [col.strip() for col in select.split(",")]
+            select = {col: 1 for col in cols}
+            query = [{"$project": select}]
             authors = Author.objects.all()
-            self.assertSuccessfulRawQuery(Author, query, authors)
+            self.assertSuccessfulMongoRawQuery(Author, query, authors)
 
     def test_translations(self):
         """
         Test of raw query's optional ability to translate unexpected result
         column names to specific model fields
         """
-        query = (
-            "SELECT first_name AS first, last_name AS last, dob, id "
-            "FROM raw_query_author"
-        )
+        query = [
+            {
+                "$project": {
+                    "first": "$first_name",
+                    "last": "$last_name",
+                    "dob": 1,
+                    "id": 1,
+                }
+            }
+        ]
         translations = {"first": "first_name", "last": "last_name"}
         authors = Author.objects.all()
-        self.assertSuccessfulRawQuery(Author, query, authors, translations=translations)
+        self.assertSuccessfulMongoRawQuery(
+            Author, query, authors, translations=translations
```

Comment:
Does the `translations` argument do anything? If I comment it out here, the test passes which indicates this isn't a proper regression test.

Reply:
No, I removed it and moved the translations to MQL.